models/articles.models.js

```diff
@@ -24,7 +26,20 @@ exports.getArticles = () => {
     GROUP BY articles.article_id
     ORDER BY articles.created_at DESC
     ;`)
-    .then(({rows}) => {
-        return rows
-    })   
-}
\ No newline at end of file
+    .then(({ rows }) => {
+      return rows;
+    });
+};
+
+exports.patchVote = (article_id, inc_votes) => {
+return db.query(`
+UPDATE articles
+SET votes = votes + $1
+WHERE articles.article_id = $2
+RETURNING *
+`, [inc_votes, article_id])
+.then(({rows}) => {
+    const article = rows[0]
+    return article
+})
```

Comment:
This is great, here is where I would include the check for `if (rows.length === 0)` which then saves you from having to make two queries to the database 💡